feature-libs/asm/components/customer-selection/customer-selection.component.html

```diff
@@ -57,7 +57,15 @@
       results.entries.length <= 0
     "
   >
-    {{ 'asm.customerSearch.noMatch' | cxTranslate }}
+    <ng-container *cxFeatureLevel="'!6.1'">
```

Comment:
Please add a TODO command to remove the feature flag associated with the Jira ticket number.

Reply:
done